contracts/ValidatorRegistry.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: BSL 1.1
+pragma solidity ^0.8.15;
+
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+
+contract ValidatorRegistry is Ownable {
+
+    uint256 public minStake;
+    uint256 public unstakePeriodBlocks;
+
+    constructor(uint256 _minStake, uint256 _unstakePeriodBlocks) {
```

Comment:
Whoops need to pass in owner account to constructor, explicitly pattern matching other `Ownable` contracts in this repo